packages/toggle-switch-react/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { ReactNode } from "react";
+
+interface Props {
```

Comment:
Shouldn't there be an onChange prop?

Reply:
Probably, I work out from checkbox, and it does not have it. But yeah, it should be easy to hook into from a parent component

Reply:
Huh, probably just a slip-up in checkbox as well. In my mind both will be used as controlled inputs more often than not